README.md

```diff
@@ -21,7 +21,7 @@ Then just call your plugin directly in an existing application:
 
     $ rebar3 format
 
-This will format every Erlang file under `/src` by default. You can specify the directory/file to format as following:
+This will format every Erlang file (including `hrl`s, `erl`s and `app.src`s) under `src`, `incude` and `test` by default, together with every `.config` file in your root directory. You can specify the directory/file to format as following:
```

Comment:
it should be `include` instead of `incude`

Reply:
🤦